eahub/localgroups/admin.py

```diff
@@ -45,41 +62,104 @@ class Meta:
             "other_info",
         ]
 
+    @classmethod
+    def widget_from_django_field(cls, f, default=widgets.Widget):
+        if isinstance(f, ArrayField):
+            return EnumArrayWidget
+        else:
+            return super().widget_from_django_field(f)
+
+    def import_row(
+        self,
+        row,
+        instance_loader,
+        using_transactions=True,
+        dry_run=False,
+        raise_errors=False,
+        **kwargs,
+    ):
+        row = self.remove_chars_from_row_keys(row, "\ufeff")
+        return super().import_row(
+            row, instance_loader, using_transactions, dry_run, raise_errors, **kwargs
+        )
+
     def before_import_row(self, row: dict, **kwargs) -> dict:
-        row["local_group_type"] = self.hydrate_local_group_type(row["type"])
-        row["local_group_types"] = self.hydrate_local_group_types(row["types"])
-        return super().before_import_row(row, **kwargs)
+        row["local_group_types"] = [
+            type
+            for type in self.hydrate_local_group_types(row["types"])
+            if type is not None
+        ]
+        organisers_users, organisers_non_users = self.hydrate_organisers(row)
+        row["organisers"] = ",".join(map(lambda x: str(x.id), organisers_users))
+        row["organisers_freetext"] = ",".join(organisers_non_users)
```

Comment:
wasn't the free text field supposed to be used for summaries? if yes perhaps we can leave a `# TODO` here to remove this code for future imports?

Reply:
no, that's for organisers who don't have a profile.
Summary is other_info field.